.github/renovate.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "automerge": false,
+  "baseBranches": [
+    "main"
+  ],
+  "branchPrefix": "renovate-",
+  "dependencyDashboard": false,
+  "extends": [
+    "config:recommended"
+  ],
+  "hashedBranchLength": 24,
+  "labels": [
+    "dependency-update"
+  ],
+  "major": {
+    "automerge": false,
+    "labels": [
+      "dependency-update",
+      "major-dependency-update"
+    ]
+  },
+  "prHourlyLimit": 10,
```

Comment:
Can we limit the number of PRs to one per day?

Reply:
I have kept the value same as used in standard and used by other team. 
Our job will be running every day at 11 PM, so hardly we will have many PRs, but just to avoid if we miss to approve and merge any PR keeping very low number might fail future jobs too.
10 is not too huge number but a reasonable keeping in mind our job will be running once per day.